# Generated by Django 4.2.13 on 2024-06-06 19:18

import django.contrib.auth.validators
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('profiles', '0006_profile_promotional_code'),
        ('users', '0007_alter_user_managers_user_covenant_id_user_user_group'),
    ]

    operations = [
        migrations.AddField(
            model_name='user',
            name='profile',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='profiles.profile', verbose_name='Profile'),
        ),
        migrations.AddField(
            model_name='user',
            name='username',
            field=models.CharField(default='df', error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
            preserve_default=False,
        ),
        migrations.AlterField(
            model_name='user',
            name='covenant_id',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='created_by_%(class)s_related', to='profiles.profile'),
        ),
    ]
